internal/task/task/common/sync_config.go

```diff
@@ -91,11 +91,11 @@ func newToolV2Mutate(dc *topology.DeployConfig, delimiter string, forceRender bo
 
 		// replace variable
 		value, err = dc.GetVariables().Rendering(value)
-		if err != nil {
-			return
+		if err == nil {
+			out = fmt.Sprintf("%s%s %s", key, delimiter, value)
+		} else { // FIX : add replace warring log
+			err = nil
```

Comment:
Ditto.

Reply:
Please refer to the discussion [here.](https://github.com/opencurve/curveadm/pull/344#discussion_r1393567785) Perhaps I need to modify the logic of sync_config? Or is there any other more suitable method?

Reply:
Yes, if render failed, you can open another pull request to fix it.